main/ui/src/main/java/org/cryptomator/ui/fxapp/FxApplication.java

```diff
@@ -68,6 +68,8 @@
 		this.licenseHolder = licenseHolder;
 		this.visibleWindows = Stage.getWindows().filtered(Window::isShowing);
 		this.hasVisibleWindows = Bindings.isNotEmpty(visibleWindows);
+
+		vaultService.setVaultRevealer(p -> this.getHostServices().showDocument(p.toUri().toString()));
```

Comment:
remove this, always pass the revealer to reveal(...) when it is called. Note that the revealer might be called (by auto unlock - which is debatable), before the FX Application is running. In that case we need to use a different revealer.

Reply:
Implemented in d5eb84a.

For the TrayMenu, i added a new revealer by making AWT calls. Not the most elegant one, but otherwise i believe it would need a big refactoring since the application is started after the tray.